### DISCUSSION TITLE
ReactiveUI class disappeared after upgrading

### DISCUSSION THREAD

**Discussion by huong-i3** [reactiveui/ReactiveUI#3646] [Q&A] [answered]
I'm migrating a Xamarin.iOS project to .Net6. I also upgraded ReactiveUI.Fody Nuget package from 9.8.1 to 19.5.1. The class ReactiveViewController that I was using before now throwing CS0246 error.

```
[CS0246](classifiedtextelementview-invokeaction:0): The type or namespace name 'ReactiveViewController' could not be found (are you missing a using directive or an assembly reference?)
```
I checked Assembly browser and it wasn't there anymore. However I checked ReactiveUI documentation it is not deleted or changed. What am I doing wrong?

**Comment by glennawatson** [comment, marked as answer]
As discussed, .NET 7 is the only supported version officially supported by Microsoft and we match Maui support policy. See https://dotnet.microsoft.com/en-us/platform/support/policy/maui
